src/content/nft/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Non-fungible tokens (NFT)
+description: An overview of NFTs on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/eth2/core.png
+summaryPoints:
+  [
+    "A way to turn anything unique into an Ethereum-based asset.",
+    "NFTs are giving more power to content creators than ever before.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+NFTs are currently taking the digital art and collectibles world by storm. Digital artists are seeing their lives change thanks to huge sales to a new crypto-audience. And celebrities are joining in as they spot a new opportunity to connect with fans.
```

Comment:
It'd be nice to add some hard data to this, e.g. according to this site "Total Crypto Art Value" is now over $100M: https://cryptoart.io/data

Reply:
I feel like Josh's article does a good job of putting numbers behind some examples: https://jjmstark.medium.com/the-year-in-ethereum-2020-98123e5f160d

Reply:
I've added a link to that cryptoart site in the further reading section. I don't want to add it to the intro for a couple reasons:
- we're not just talking about art
- any figures we pull out will have to be updated regularly